src/utils/utils.ts

```diff
@@ -57,9 +57,10 @@ Object.assign(globalThis, { dispatchCustomEvents })
 export function selectElementContents(el: Node) {
     const range = document.createRange()
     range.selectNodeContents(el)
-    const sel = window.getSelection()!
+    const sel = globalThis.getSelection()!
```

Comment:
be aware this might break on Firefox if you didn't add it in public/firefoxFix.js

Reply:
Added. There is a change on `public/firefoxFix.js` in this PR.